docs/mosviz/notebook.rst

```diff
@@ -15,3 +15,22 @@ After running the code above, you can interact with the Mosviz application from
 subsequent notebook cells via the API methods attached to the `mosviz` object,
 for example loading data into the app as described in :ref:`mosviz-import-data`.
 
+.. seealso::
+
+    `Cubeviz data export <https://jdaviz.readthedocs.io/en/latest/cubeviz/notebook.html>`_
+        Cubeviz documentation on data exporting.
+
+The `~jdaviz.configs.mosviz.helper.MosViz` helper class can be used similarly to how
+`cubeviz` is used in the previous link.
+The viewers in `mosviz` that can be used that way are `image-viewer`, `spectrum-viewer`,
+and `spectrum-2d-viewer`.
+
+It is also possible to extract data from the `table-viewer` using the following code::
```

Comment:
To me, "extract data" is a little ambiguous and suggests you can extract specific data out of the table? I don't think this is possible? I think something like:
```suggestion
It is also possible to extract the contents from the `table-viewer` via:
```

Reply:
Good point, I fixed that and added information about the `to_csv` parameters in the most recent commit.